personalInfoDashboard.js

```diff
@@ -0,0 +1,73 @@
+// information
+const firstName = "Mariam";
+const lastName = "Poghosyan";
+const birthYear = 2003;
+const currentYear = 2025;
+const isStudent = false;
+const hobbies = ['listening to music', 'hiking', 'Armenian folk dances']
+let contact = {
+  email: 'mariam.poghosyan.techx@gmail.com',
+  phone: '+374 93 22-07-67',
+  city: 'Yerevan',
+};
+
+console.log(`Hi, my name is ${firstName} ${lastName}. I live in ${contact.city}.`);
+console.log(`I am ${currentYear - birthYear} years old and currently ${isStudent ? "a student" : "not a student"}.`);
+
+//  Type Coercion & Comparison
```

Comment:
you're using 2 spaces here, 1 is enough
```suggestion
// Type Coercion & Comparison
```

Reply:
thanks, I changed it.